scripts/common.lib.ps1

```diff
@@ -127,7 +127,7 @@ function Locate-VsInstallPath($hasVsixExtension ="false"){
   $requiredPackageIds = @()
 
   $requiredPackageIds += "Microsoft.Component.MSBuild" 
-  $requiredPackageIds += "Microsoft.Net.Component.4.6.TargetingPack"
+  $requiredPackageIds += "Microsoft.Net.Component.4.5.2.TargetingPack"
```

Comment:
I am not sure about this, but we target v4.5.2 but v4.6 in our projects. Let's say if a user has the v4.6 targeting pack installed but not v4.5.2 won't we fail anyway in that case?

Reply:
I think I confused this with the 4.6 Portable pack. I looked into my VS installation and I have all targeting packs installed, to avoid those issues, so no problem for me. Ignore this then.